src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,380 @@
+    variable     =  "{" var-name "}"
+    PAYID-EASY-CHECKOUT-URI-Template =  *( uri-char / variable )
+
+For example:
+
+    Input:    alice$wallet.com
+              amount = 10
+              receiverPayID=pay$merchant.com
+              currency=XRP
+              network=XRPL
+              nextUrl=https://merchant.com/thankyou
+    Template: https://wallet.com/checkout?amount={amount}&receiverPayId={receiverPayID}&currency={currency}&network={network}&nextUrl={nextURL}
+    Output:   https://wallet.com/checkout?amount=10&receiverPayId=payid%2Apay%24merchant.com&currency=XRP&network=XRPL&nextUrl=https://merchant.com/thankyou
+
+TODO: Should we define acceptable URL template variable values for the redirect?
+
+## PayID Easy Checkout URL Assembly
+The PayID Easy Checkout URL is constructed by expanding the PayID Easy Checkout URI Template as defined in section 3 of
+[RFC6570][] buy applying values corresponding to the variables specified in
+section (TODO: link to template section) to the Template.
+
+The PayID Checkout URI template MAY not contain the complete set of variables specified in section (TODO link section) 
+of this document.  However, PayID Easy Checkout clients MUST replace each variable present in the URI Template with a value. 
+PayID Easy Checkout clients MAY replace URI Template values with an empty string, however it is RECOMMENDED that each
+variable be replaced with a non-empty value.
+
+The result of expanding the PayID Easy Checkout URI Template is a PayID Easy Checkout URL.
+This URL SHOULD represent a resource that the customer can use to complete a payment. As previously stated, the
+ways in which the customer and merchant use that resource is outside the scope of this protocol.
+The PayID Easy Checkout URL SHOULD be parsed to retrieve the values set by the recipient client. It is RECOMMENDED
+that wallet UIs use these values to pre-populate a payment transaction.
+
+# PayID Easy Checkout JRDs
+ This section defines the PayID Easy Checkout Link, which conforms to section 4.4 of the
+ Webfinger RFC.  In order for a PayID server to enable PayID Easy Checkout, a PayID Discovery query to the server
+ MUST return a JRD containing a PayID Easy Checkout Link.
+ 
+ The Link MUST include the Link Relation Type of section (TODO: link to link type section) in the object's 'rel' field.
+ The Link MUST also include a PayID Easy Checkout URI Template in the 'template' field of the link.
+ 
+    * 'rel': `https://payid.org/ns/payid-easy-checkout-uri/1.0`
+    * 'template': A PayID Easy Checkout URI Template
+    
+ The following is an example of a PayID Easy Checkout Link that indicates a PayID Easy Checkout URI Template:
+ 
+    {
+        "rel": "https://payid.org/ns/payid-easy-checkout-uri/1.0",
+        "template": https://wallet.com/checkout?amount={amount}&receiverPayId={receiverPayID}&currency={currency}&network={network}&nextUrl={nextURL}
+    }
+
+# Security Considerations
```

Comment:
@aanchal4 I wrote down some initial security thoughts, but could use your expertise here if you're available